mkdocs/docs/HPC/rhel9.md

```diff
@@ -0,0 +1,158 @@
+
+(or another toolchain with a year-based version older than `2023a`)
+
+The `module` command will produce a clear warning when you are loading modules
+that are using a toolchain that will no longer be available after the cluster
+has been migrated to RHEL 9.
+For example:
+
+```
+foss/2022b:
+   ___________________________________
+  /  This module will soon no longer  \
+  \  be available on this cluster!    /
+   -----------------------------------
+         \   ^__^
+          \  (xx)\_______
+             (__)\       )\/\
+              U  ||----w |
+                 ||     ||
+
+Only modules installed with a recent toolchain will still be available
+when this cluster has been migrated to the RHEL 9 operating system.
+Recent toolchains include GCC(core)/12.3.0, gompi/2023a, foss/2023a,
+iimpi/2023a, intel/2023a, gfbf/2023a, and newer versions.
+
+You should update your workflow or job script to use more recent software
+installations, or accept that the modules you currently rely on will soon
+no longer be available.
+
+To request a more recent version of the software you are using,
+please submit a software installation request via:
+
+https://www.ugent.be/hpc/en/support/software-installation-request
+
+The HPC-UGent Tier-2 clusters running RHEL 8 will be migrated to RHEL 9.
+
+For more information, see https://docs.hpc.ugent.be/rhel9/
+
+If you have any questions, please contact hpc@ugent.be .
+```
+
+If you require software that is currently only available with an older toolchain
+on the HPC-UGent Tier-2 clusters that are still running RHEL 8,
+check via `module avail` if a more recent version is installed that you can switch to,
+or submit a [software installation request](https://www.ugent.be/hpc/en/support/software-installation-request)
+so we can provide a more recent installation of that software which you can adopt.
+
+It is a good idea to test your software on the `shinx` cluster,
+which is already running RHEL 9 as operating system,
+to be sure if it still works.
+We will provide more RHEL 9 nodes on other clusters to test on soon.
+
+## Planning
+
+We plan to migrate the HPC-UGent Tier-2 clusters that are still
+using RHEL 8 to RHEL 9 one by one, following the schedule outlined below.
+
+| ***cluster*** | ***migration start*** | ***migration completed on*** |
+| --- | ---- | --- |
+| `skitty` | Monday 30 September 2024 | |
+| `joltik` | October 2024 | |
+| `accelgor` | November 2024 | |
+| `gallade` | December 2024 | |
+| `donphan` | February 2025 | |
+| `doduo` (default cluster) | February 2025 | |
+| login nodes switch | February 2025 | |
+
+Migration the `donphan` and `doduo` clusters to RHEL 9 and switching `login.hpc.ugent.be` to RHEL 9 login nodes
+will be done at the same time.
+
+We will keep this page up to date when more specific dates have been planned.
+
+!!! warning: This planning below is subject to change, some clusters may get migrated
+later than originally planned.
+
+**Please check back regularly.**
```

Comment:
@wdpypere Warning must be indented, or it won't render correctly:
```suggestion
!!! warning
    This planning below is subject to change, some clusters may get migrated later than originally planned.

    **Please check back regularly.**

Reply:
if the markdown is not really markdown, we should disable the markdown previews